src/pages/settings/PaymentsPage.js

```diff
@@ -92,7 +92,7 @@ class PaymentsPage extends React.Component {
                     <FixedFooter>
                         <Button
                             success
-                            isDisabled={!this.state.payPalMeUsername}
+                            isDisabled={Boolean(this.props.payPalMeUsername)}
```

Comment:
Why do we need to wrap it in `Boolean`?

Reply:
Otherwise, there's a warning saying we're trying to pass a string into a prop that's expects a boolean. If you don't agree with this I can always get rid of it.
```suggestion
                            isDisabled={this.props.payPalMeUsername}
```

Reply:
Oh, whoops I didn't see that was a String! Makes total sense.